docs/pages/access-controls/access-request-plugins/servicenow.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Access Requests with ServiceNow
+description: How to set up Teleport's ServiceNow plugin for privilege elevation approvals.
+---
+
+With Teleport's ServiceNow integration, engineers can access the infrastructure
+they need to resolve incidents quickly, without longstanding admin permissions
+that can become a vector for attacks.
+
+Teleport's ServiceNow integration allows you to treat Teleport Role Access
+Requests as ServiceNow incidents, notify the appropriate on-call team, and
+approve or deny the requests via Teleport.
+
+This guide will explain how to set up Teleport's Access Request plugin for
+ServiceNow.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- An ServiceNow account with access to read and write to and from the 'incident' table.
+- The ServiceNow integration is currently only available in Teleport Cloud.
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/4. Create services
+
+We will configure the ServiceNow plugin to create an incident when
+certain users create an Access Request.
+
+## Step 2/4. Define RBAC resources
+
+The Teleport ServiceNow plugin works by receiving Access Request events from the
+Teleport Auth Service and, based on these events, interacting with the ServiceNow
+API.
+
+### Create a requester role
+
+To create a user first navigate to Management -> Access -> Roles
```

Comment:
Don't we have a preset requester role now? If so, I think we should rewrite this to use the preset, instead of creating another one.

I would also remove the screenshot, the text description of the YAML is enough.

Reply:
What about changing it like this so it doesn't overwrite the preset role and creates one with less scope for this demo?

Reply:
This is okay, but in the past Sasha has asked us to use the preset roles wherever possible to make the getting started experience as simple as we can.

If we already have a preset that includes the appropriate permissions, wouldn't it be much simpler to just use it?

Reply:
@EdwardDowling Zac has a good point: we already have a builtin `requester` role that allows users to request `access` role. Let's just use it here.